lib/game.rb

```diff
@@ -12,42 +12,94 @@ def initialize
   def welcome
     start = ""
     puts "Welcome to BATTLESHIP\nEnter p to play. Enter q to quit."
-    until start == "p" || start =="q" do
+    until start == "p" || start == "q" do
       print "> "
-      start = gets.chomp
+      start = gets.chomp.downcase
       puts "Please select valid option 'p' or 'q'."
     end
 
-    if start = "p"
-      @player_1.place_ship(player_1.cruiser)
-      @player_1.place_ship(player_1.submarine)
-      puts "I have laid out my ships on the grid."
-      @player_2.board.render
-      ask_for_placement(@player_2.cruiser)
-      @player_2.board.render(true)
-      ask_for_placement(@player_2.submarine)
-      @player_2.board.render(true)
-    else
-      puts "Goodbye!!"
-    end
+    # if start == "p"
+    #   @player_1.place_ship(player_1.cruiser)
+    #   @player_1.place_ship(player_1.submarine)
+    #   puts "I have laid out my ships on the grid."
+    #   @player_2.board.render
+    #   ask_for_placement(@player_2.cruiser)
+    #   @player_2.board.render(true)
+    #   ask_for_placement(@player_2.submarine)
+    #   @player_2.board.render(true)
+    # else
+    #   puts "Goodbye!!"
+    # end
+  end
+
+  def setup
+    @player_1.place_ship(player_1.cruiser)
+    @player_1.place_ship(player_1.submarine)
+    puts "I have laid out my ships on the grid."
+    puts @player_2.board.render
+    ask_for_placement(@player_2.cruiser)
+    puts @player_2.board.render(true)
+    ask_for_placement(@player_2.submarine)
+    puts @player_2.board.render(true)
   end
 
   def play
+    puts ('=' * 10) + "COMPUTER BOARD" + ('=' * 10)
+    puts @player_1.board.render
+    puts " "
+    puts ('=' * 10) + "PLAYER BOARD" + ('=' * 10)
+    puts @player_2.board.render(true)
+
     until player_1.dead? || player_2.dead?
+
+      # @player_1.board[@player_2.turn].fire_upon
+      # puts ('=' * 10) + "COMPUTER BOARD" + ('=' * 10)
+      # puts @player_1.board.render
+      # puts " "
+      # puts ('=' * 10) + "PLAYER BOARD" + ('=' * 10)
+      # puts @player_2.board.render(true)
+      # @player_1.dead?
+
+      puts "I am firing on you"
+      @player_2.board.cells[comp_turn].fire_upon
```

Comment:
I would still like to try and get the "Firing on you" to say with cell is being fired on.